Sources/Logging/LogHandler.swift

```diff
@@ -151,3 +159,27 @@ public protocol LogHandler {
     ///        `LogHandler`.
     var logLevel: Logger.Level { get set }
 }
+
+extension LogHandler {
+    @available(*, deprecated, message: "You should implement this method instead of using the default implementation")
+    public func log(level: Logger.Level,
```

Comment:
Will this do the right thing? Maybe I'm misunderstanding how this works...?

Reply:
These are the default implementations if you do not provide them. We have to implement both as the respective other. Because we could have an old `LogHandler` (which lacks the new `log` method) but the old `log` method is also public API so we have to continue supporting it...

Reply:
I was one line off with my question (was about the deprecated, how it'll work) but now I see, thanks for reiterating.

I guess the deprecation would not really show up except in perhaps if someone tested their log handler and called log on it, then that'd notice it's the deprecated one hm... It would not warn end users since it's the handler after all.

I guess won't hurt to have the deprecated on it though 👍 🤔